e2e/utils.go

```diff
@@ -31,17 +31,18 @@ import (
 	testutils "k8s.io/kubernetes/test/utils"
 )
 
+/* #nosec .*/
```

Comment:
needs a comment why it is disabled, just `.` is not an explanation

Reply:
Sure. `.` was added just as `Top level comments need to end in a period`. I'll add an explanation too.